aclk/aclk.c

```diff
@@ -813,6 +813,7 @@ struct label *add_aclk_host_labels(struct label *label) {
             proxy_str = "none";
             break;
     }
+    label = add_label_to_list(label, "_aclk_impl", "Next Generation", LABEL_SOURCE_AUTO);
```

Comment:
I don't know if it would be problematic to have spaces in labels (in general)

Reply:
should not be a problem if it is in value